packages/twenty-front/src/modules/settings/components/SettingsNavigationDrawerItems.tsx

```diff
@@ -109,7 +106,7 @@ export const SettingsNavigationDrawerItems = () => {
     return matchPath(
       {
         path: pathName,
-        end: !accountSubSetting.matchSubPages,
+        end: true,
       },
```

Comment:
Hey @FelixMalfait! I'm thinking this change will really make the code more intentional, don't you agree? Or do you think we should revert it? I mean, it doesn't really make a difference unless we want matchSubPages as false for one of the Emails or Calendars, but I highly doubt that's the case. What do you think?

Reply:
I guess we can use the same pattern we used at the above level = default to true but leave it as an option that can be configured. Just did a quick edit